tests/contrib/tracking/test_em.py

```diff
@@ -63,23 +63,23 @@ def model(detections, args):
 
 
 # This should match detection_model's existence part.
-def exists_log_likelihood(objects, args):
+def compute_exists_logits(objects, args):
```

Comment:
The changes in this file work around https://github.com/pytorch/pytorch/issues/9977

Reply:
FYI @alicanb you may need to make similar changes to get tracking models to work in 0.4.1 (i.e. avoid constructing a `torch.stack`ed or `torch.cat`'d tensor). Or you could simply avoid upgrading.